tests/chart_tests/test_houston_configmap.py

```diff
@@ -198,6 +199,7 @@ def test_houston_configmapwith_loggingsidecar_enabled_with_overrides():
         "enabled": True,
         "name": sidecar_container_name,
         "terminationEndpoint": terminationEndpoint,
+        "customConfig": False,
```

Comment:
Can we add a test where we expect `"customConfig": True,` ? Would that even make sense in this case?

Reply:
Added. Thanks @pgvishnuram